### PR TITLE
ETFE-2786 Get excise product codes & country codes

### PR DESCRIPTION
**Pull request by grahampaulcook**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -69,6 +58,11 @@ microservice {
       host = localhost
       port = 8311
     }
+    emcs-tfe-reference-data {
+      protocol = http
+      host = localhost
+      port = 8312
+    }
```
Comment (LewisMitchell) [inline conf/application.conf:65]: There is an entry already for `reference-data`. Could you remove that entry please?

Comment (LewisMitchell) [reply conf/application.conf:65]: (and replace any occurrences of `reference-data` with `emcs-tfe-reference-data`)

File: it/uk/gov/hmrc/emcstfefrontend/support/IntegrationBaseSpec.scala [hunk 2]
```diff
@@ -24,17 +24,21 @@ import play.api.libs.json.{JsValue, Json}
...
 trait IntegrationBaseSpec extends SessionCookieBaker with UnitSpec with WireMockHelper with GuiceOneServerPerSuite
   with BeforeAndAfterEach with BeforeAndAfterAll {
 
+  implicit lazy val ec: ExecutionContext = app.injector.instanceOf[ExecutionContext]
+
   lazy val client: WSClient = app.injector.instanceOf[WSClient]
 
   def servicesConfig: Map[String, _] = Map(
     "microservice.services.auth.port" -> WireMockHelper.wireMockPort,
     "microservice.services.reference-data.port" -> WireMockHelper.wireMockPort,
```
Comment (LewisMitchell) [inline it/uk/gov/hmrc/emcstfefrontend/support/IntegrationBaseSpec.scala:38]: Duplicate reference data config entry, could you remove please?

File: test/uk/gov/hmrc/emcstfefrontend/base/SpecBase.scala
Comment (LewisMitchell) [inline test/uk/gov/hmrc/emcstfefrontend/base/SpecBase.scala:1]: Should all specs in `test` extend this trait now?

File: test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetExciseProductCodesConnector.scala
Comment (LewisMitchell) [inline test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetExciseProductCodesConnector.scala:1]: Should we also have a MockGetExciseProductCodesService?

File: test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetMemberStatesConnector.scala
Comment (LewisMitchell) [inline test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetMemberStatesConnector.scala:1]: Should we also have a MockGetMemberStatesService?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (grahampaulcook) [reply conf/application.conf]: Fixed in https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/903817e690ffef14174a5b82fddc305d93662fd6

File: test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetExciseProductCodesConnector.scala
Comment (grahampaulcook) [reply test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetExciseProductCodesConnector.scala:1]: https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/ba5c97ec53be30010b08092b6fcbd2f94f2ba543

File: test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetMemberStatesConnector.scala
Comment (grahampaulcook) [reply test/uk/gov/hmrc/emcstfefrontend/mocks/connectors/MockGetMemberStatesConnector.scala:1]: https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/ba5c97ec53be30010b08092b6fcbd2f94f2ba543

File: it/uk/gov/hmrc/emcstfefrontend/support/IntegrationBaseSpec.scala (hunk 2 above)
Comment (grahampaulcook) [reply it/uk/gov/hmrc/emcstfefrontend/support/IntegrationBaseSpec.scala]: https://github.com/hmrc/emcs-tfe-frontend/pull/40/commits/db22e64b51e054f0e637067ede327ab02e2f502d

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**